### DISCUSSION TITLE
Cannot read properties of null (reading 'writeFile')

### DISCUSSION THREAD

**Discussion by shoopdawoop** [requarks/wiki#4885] [Error / Bug Report]
So I get a "Cannot read properties of null (reading 'writeFile')" error whenever I edit a page and apply changes.

I then hit Conflict, use remote version, confirm, then hit cancel (red X) to leave the edit mode (discard changes),

and then the change has indeed been saved. Every time :)

I am running the latest v2.5.268 as a docker container with postgres:14-alpine and I tested this in Brave Browser and Safari Browser.

Anyone got an idea what might be causing this error?

**Comment by DevLomoSE** [comment]
im using 2.5.303 and seeing the same "issue" / bug, something else i've noticed is that this only happens when there is a change, if i try to edit then click on close there is no error, only when there is a change that i want to apply.
